package/tests/burner.cpp

```diff
@@ -41,33 +39,49 @@ double burn_for(float ms_interval = 1.0) {
   return burn_result;
 }
 
-int main(int argn, char *argv[]) {
+int main(int argc, char *argv[]) {
   float runtime{};
   unsigned int threads{}, procs{};
+  int do_help{0};
+
+  static struct option long_options[] = {
+      {"threads", required_argument, NULL, 't'},
+      {"procs", required_argument, NULL, 'p'},
+      {"time", required_argument, NULL, 'r'},
+      {"help", no_argument, &do_help, 1},
+      {0, 0, 0, 0}
+  };
+
+  char c;
+  while ((c = getopt_long(argc, argv, "t:p:t:h", long_options, NULL)) != -1) {
+    switch (c) {
+    case 't':
+      threads = std::stoi(optarg);
+      break;
+    case 'p':
+      procs = std::stoi(optarg);
+      break;
+    case 'r':
+      runtime = std::stof(optarg);
+      break;
+    case 'h':
+      do_help = 1;
+      break;
+    }
+  }
 
-  prog_opt::options_description desc(
-      "burner is a simple cpu burner program that can run in multiple threads\n"
-      "and/or processes.\n\n"
-      "If both threads and procs are sepecified then each process runs\n"
-      "multiple threads (so the load is threads * procs).\n\n"
-      "Allowed options:");
-  desc.add_options()
-      ("help,h", "print this help message")
-      ("threads,t", prog_opt::value<unsigned int>(&threads)->default_value(1),
-          "run in N threads (setting 0 will used the hardware concurrency value)")
-      ("procs,p", prog_opt::value<unsigned int>(&procs)->default_value(1),
-          "run in N processes (setting 0 will used the hardware concurrency value)")
-      ("time,r", prog_opt::value<float>(&runtime)->default_value(default_runtime), "run for T seconds")
-  ;
-
-  // Parse command line
-  prog_opt::variables_map var_map;
-  prog_opt::store(prog_opt::parse_command_line(argn, argv, desc), var_map);
-  prog_opt::notify(var_map);
-
-  if (var_map.count("help")) {
-      std::cout << desc << "\n";
-      return 1;
+  if (do_help) {
+    std::cout <<
+        "burner is a simple cpu burner program that can run in multiple threads\n"
+        "and/or processes.\n\n"
+        "If both threads and procs are sepecified then each process runs\n"
+        "multiple threads (so the load is threads * procs).\n" << std::endl;
+    std::cout << "Options:\n"
+        << " [--threads, -t N]  Number of threads to run (default 1)\n"
+        << " [--procs, -p N]    Number of processes to run (default 1)\n"
```

Comment:
A minor point but the default values for threads and processes are set to the hardware concurrency value instead of 1. Maybe we should document as such.